tw2/jqplugins/jqgrid/widgets/sqla.py

```diff
@@ -123,7 +122,7 @@ def _get_properties(cls):
             def relation_sorter(x, y):
                 xname = dotted_getattr(x, 'direction.name', 'a')
                 yname = dotted_getattr(y, 'direction.name', 'a')
-                return -1 * cmp(xname, yname)
+                return (xname < yname) - (xname > yname)
```

Comment:
How about a short comment that it's based on the old `cmp`?

Reply:
done